linked_list/node.py

```diff
@@ -0,0 +1,7 @@
+class Node(object):
+    def __init__(self, val, next=None):
+        self.val = val
+        self._next = next
+
+    def __repr__(self):
+        return '{val}'.format(val=self.val)
```

Comment:
Since the only thing in the string you are returning is the value you could just return `str(self.val)` instead.